CMakeLists.txt

```diff
@@ -528,6 +551,7 @@ set(server_lib_deps
   ${_REFLECTION}
   ${_UTF8CPP}
   ${CMAKE_DL_LIBS}
+  ni_grpc_sideband
```

Comment:
This would change to `_GRPC_SIDEBAND` once you address other comment towards top of this file related to `USE_SUBMODULE_LIBS`

Reply:
Updated